src/components/ItaliaTheme/Blocks/Common/SearchFilters/SelectFilter.jsx

```diff
@@ -52,15 +53,20 @@ const SelectFilter = ({
         );
       }
     } else if (options.vocabulary) {
-      dispatch(getVocabulary({ vocabNameOrURL: options.vocabulary }));
+      dispatch(
+        getVocabulary({
+          vocabNameOrURL: options.vocabulary,
+          size: optionsQuerySize,
```

Comment:
@sabrina-bongiovanni non ho capito se optionsQuerySize è 25 fisso oppure lo puoi passare alto a piacere.
Perchè è proprio il caso in cui scrivere 1000 per avere tutti i luoghi.
O è il -1 che vedo sotto che fa la magia?

Reply:
@luca-bellenghi il -1 fa la magia di eliminare il limite di paginazione!
quindi avevo bisogno di passare questo valore come parametro per poter eliminare, o modificare, questo parametro dove serve. Per settare un default che non rompesse tutti gli altri blocchi di ricerca, ho messo il valore 25 che di base è quello che Plone imposta comunque sotto il cofano. Così i blocchi già esistenti non si rompono e rimangono invariati, ma se da un blocco si vuole cambiare questo parametro, bassa passarlo come prop